src/main/scala/codacy/plugins/test/checkstyle/CheckstyleImplicits.scala

```diff
@@ -0,0 +1,12 @@
+package codacy.plugins.test.checkstyle
+
+import scala.xml.NodeSeq
+
+object CheckstyleImplicits {
+  implicit class NodeSeqOps(val node: NodeSeq) extends AnyVal {
+
+    def getProperty(name: String): String = {
+      (node \ "property").find(_ \@ "name" == name).get \@ "value"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/gh/codacy/codacy-plugins-test/pullRequest?prid=4860117)